src/main/java/nl/tudelft/lifetiles/graph/models/jgrapht/JGraphTGraphAdapter.java

```diff
@@ -69,6 +74,7 @@ public final boolean addEdge(final V source, final V destination) {
         try {
```

Comment:
Using a `try .. catch` block instead of pre-checking with `contains` is bad code style and also not very *fail fast*.